extensions/data-transfer/portability-data-transfer-google/build.gradle

```diff
@@ -37,6 +38,10 @@ dependencies {
     compile("com.googlecode.ez-vcard:ez-vcard:${ezVcardVersion}")
     compile("com.google.apis:google-api-services-plus:${googlePlusVersion}")
 
+    // Needed for OpenSocial ActivityStreams
+    compile group: 'joda-time', name: 'joda-time', version: '2.3'
```

Comment:
Do we want to make the version a constant in the global version file?

Reply:
No, Activity stream depends on this specific version of joda-time, So I don't think we want to generalize it.

Reply:
Ok, let's add a comment